presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -582,6 +583,12 @@ public Void visitSpatialJoin(SpatialJoinNode node, Void context)
             return null;
         }
 
+        @Override
+        public Void visitMergeJoin(MergeJoinNode node, Void context) {
+            printNode(node, "MergeJoin", NODE_COLORS.get(NodeType.MERGE_JOIN));
```

Comment:
visit the children too similar to other join nodes.

Reply:
Good point. Updated. Thanks.